iceoryx_posh/test/moduletests/test_roudi_memory_provider.cpp

```diff
@@ -117,7 +157,6 @@ TEST_F(MemoryProvider_Test, AddMemoryBlockExceedsCapacity)
 
 TEST_F(MemoryProvider_Test, CreateWithoutMemoryBlock)
 {
-    EXPECT_CALL(sut, createMemoryMock(_, _)).Times(0);
```

Comment:
I wouldn't remove this. We need to check if our invariants hold and that is one of these.